TestModels/MultipleModels/dafny/PrimaryProject/src/Index.dfy

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 include "SimpleMultiplemodelsPrimaryprojectImpl.dfy"
 
-module {:extern "simple.multiplemodels.primaryproject.internaldafny" } SimpleMultiplemodelsPrimaryprojectService refines AbstractSimpleMultiplemodelsPrimaryprojectService {
+module PrimaryProject refines AbstractSimpleMultiplemodelsPrimaryprojectService {
```

Comment:
certainly the new name is better;
do you intend to update the other runtimes when we eventually merged `python-reviewed` into `main-1.x`?

Reply:
This is basically a placeholder to unblock Python;
I'm planning to take in whatever change Jose makes here for NET/Java.